pkgs/tools/networking/gnirehtet/paths.patch

```diff
@@ -33,3 +33,16 @@ index 4f1be53..96d2e78 100644
          Ok(output) => {
              if output.status.success() {
                  // the "regex" crate makes the binary far bigger, so just parse the versionCode
+Index: gnirehtet/relay-rust/src/adb_monitor.rs
+===================================================================
+--- gnirehtet.orig/relay-rust/src/adb_monitor.rs
++++ gnirehtet/relay-rust/src/adb_monitor.rs
+@@ -206,7 +206,7 @@ impl AdbMonitor {
+
+     fn start_adb_daemon() -> bool {
+         info!(target: TAG, "Restarting adb daemon");
+-        match process::Command::new("adb")
++        match process::Command::new("@adb@")
```

Comment:
Is it not easier and less likely to break to use `makeWrapper` instead and extend the PATH?

Reply:
Initially, the reason I went for a patch was that we need the patch anyway to fix the path to the apk.